flexget/plugins/output/pushalot.py

```diff
@@ -31,12 +31,15 @@ class OutputPushalot(object):
     Configuration parameters are also supported from entries (eg. through set).
     """
     default_body = "{% if series_name is defined %}{{tvdb_series_name|d(series_name)}} " \
-                   "{{series_id}} {{tvdb_ep_name|d('')}}{% elif imdb_name is defined %}{{imdb_name}} "\
+                   "{{series_id}} {{tvdb_ep_name|d('')}}{% elif imdb_name is defined %}{{imdb_name}} " \
```

Comment:
Can we just ditch the `\` here? Default python string concatenation or whatever it's called should take care of this, no?